src/dosbox.cpp

```diff
@@ -1240,17 +1240,19 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&MSCDEX_Init);
 	secprop->AddInitFunction(&DRIVES_Init);
 	secprop->AddInitFunction(&CDROM_Image_Init);
-#if C_IPX
+
 	secprop = control->AddSection_prop("ipx", &IPX_Init, changeable_at_runtime);
 	pbool = secprop->Add_bool("ipx", when_idle, false);
-	pbool->Set_help("Enable IPX over UDP/IP emulation (disabled by default).");
+	pbool->SetOptionHelp("Enable IPX over UDP/IP emulation (disabled by default).");
+#if C_IPX
+	pbool->SetEnabledOptions({"ipx"});
 #endif
 
-#if C_SLIRP
 	secprop = control->AddSection_prop("ethernet", &NE2K_Init, changeable_at_runtime);
 
 	pbool = secprop->Add_bool("ne2000", when_idle, true);
-	pbool->Set_help(
+	pbool->SetOptionHelp(
```

Comment:
Wouldn't the `ethernet` section be present then even if we're compiling without SLIRP support? I think the old `#if` needs to stay in this case, wrapping the whole section definition.

Reply:
Fixed it. 😄

Reply:
Very nice 😄